build.sbt

```diff
@@ -118,10 +118,8 @@ lazy val site = project.in(file("site"))
 
 // General Settings
 lazy val commonSettings = Seq(
-  scalaVersion := "2.13.1",
-  crossScalaVersions := Seq(scalaVersion.value, "2.12.11"),
+  crossScalaVersions := Seq(scala_2_12, scala_2_13),
```

Comment:
You probably need to do this as `ThisBuild / crossScalaVersions` in the root for sbt-github-actions to pick it up.